### ISSUE TITLE
HTML emails look different for sender

### ISSUE THREAD

**Issue by thstyl2000** [nextcloud/mail#6548]
### Is your feature request related to a problem? Please describe.

Formatting is somewhat "bilnd" which makes it a bit difficult to "trust" the outcome and makes the editing part not looking very nice.

### Describe the solution you'd like

To be able to see the format of the sent mail directly when editing, in a WYSIWYG mode.

**Comment by ChristophWurst** [comment]
The app uses CKEditor right now. This is WYSIWYG.

**Comment by thstyl2000** [comment]
Yes, I know that. But we don't really see what we get, for example:

![Screenshot_20220526_132618](https://user-images.githubusercontent.com/18458458/170480982-3ce13fde-2b16-4e9d-9d4d-e41c469a1e0d.png)

![Screenshot_20220526_133526](https://user-images.githubusercontent.com/18458458/170480979-5a1528cb-f267-4ac7-9f8a-d9dcb76859d9.png)

Fonts are smaller, margins different and the quote line does not appear....(Never noticed that before...Maybe it has to do with the fact that I have a white theme?)

**Comment by ChristophWurst** [comment]
Right. As far as I understand this has to do with how the HTML is rendered on the receiving end. Like a HTML list `<li>` may be indented or not. We do not specify any styling, so there is a lot of freedom for the email clients.

Now, I wonder how other email clients handle this. Do Thunderbird, Gmail & others send stricter styling for their HTML messages, so they look more consistent with what the sender sees? That needs some research.

**Comment by ChristophWurst** [comment]
Please post the source as well
